### PR TITLE
test: dxfeed integration tests

### PR DESCRIPTION
**Pull request by austinborn**
## Description
Integration tests for dxfeed adapter

## Changes

- Integration test for `price` endpoint

## Steps to Test
`yarn && yarn setup && yarn test dxfeed/test/integration`

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json [hunk 1]
```diff
@@ -23,15 +23,9 @@
...
   "dxfeed": {
-    "AAPL": "AAPL:BFX",
-    "AMZN": "AMZN:BFX",
-    "FB": "FB:BFX",
-    "FTSE": "UKX.IND:TEI",
-    "GOOGL": "GOOGL:BFX",
-    "MSFT": "MSFT:BFX",
-    "N225": "NKY.IND:TEI",
-    "NFLX": "NFLX:BFX",
-    "TSLA": "TSLA:BFX",
```
Comment (boxhock) [inline packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json:34]: Why are we changing/removing so many of these?

Comment (austinborn) [reply packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json:34]: I'm happy to take suggestions on how to fix this, but the problem is that these symbol mappings are outdated and result in errors from the api. A lot of the symbols now work without the override.

Comment (boxhock) [reply packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json:34]: Let's confirm with the data team before making any changes to these. Also, just want you to confirm that the new symbols work with our own dxFeed endpoint (not `https://tools.dxfeed.com`, which is a demo endpoint)

Comment (austinborn) [reply packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json]: Reverted, I needed to use the chainlink endpoint to successfully query these symbols

**Review by boxhock — APPROVED** [review]

**Pull request merged**